addons/web/static/tests/views/list_tests.js

```diff
@@ -2217,6 +2219,50 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+     QUnit.test('aggregates digits can be set with the currency digits', async function (assert) {
+        assert.expect(2);
+
+         this.data.foo.fields.amount.currency_field = 'currency_id';
+         this.data.res_currency.records.push({
+             id: 3,
+             display_name: 'RUB',
+             symbol: '₽',
+             position: 'after',
+             rounding: 0.0001,
+             decimal_places: 4,
+         });
+
+         this.data.foo.records.forEach(e => e.currency_id = 3);
+
+         const currencies = {};
+         this.data.res_currency.records.forEach(currency => {
+             currencies[currency.id] = {...currency};
+         });
+
+         currencies[3].digits = [69, 4];
+
+         const list = await createView({
+             View: ListView,
+             model: 'foo',
+             data: this.data,
+             arch: '<tree>' +
+                 '<field name="amount" widget="monetary" sum="Sum" />' +
+                 '<field name="currency_id" invisible="1"/>' +
+                 '</tree>',
+             session: {
+                 currencies: currencies,
+             },
+         });
+
+        assert.strictEqual(list.$('.o_data_row td:nth(1)').text(), '1200.0000' + '\xa0' + '₽',
+            "field should still be formatted based on currency");
+        assert.strictEqual(list.$('tfoot td:nth(1)').text(), '2000.0000',
+            "aggregates monetary is formatted with the currency " +
+            "digits if no digits attributes has been set");
+
+        list.destroy();
+    });
+
```

Comment:
there should be a test for the N/A case, and one for the case where there is a digits attrs on the column